waveorder/visual.py

```diff
@@ -414,7 +407,7 @@ def parallel_4D_viewer(
         raise ValueError(
             "number of titles does not match with the number of channels"
         )
-    num_row = np.int(np.ceil(N_channel / num_col))
+    num_row = int(np.ceil(N_channel / num_col))
```

Comment:
This reminds me of NEP29: the advised support range for NumPy is now 1.21+

Reply:
Thanks!